python/lsst/ts/aos/utils/scripts/lut_update.py

```diff
@@ -0,0 +1,137 @@
+import argparse
```

Comment:
Please add a standard boilerplate GPL license.

And another generic note: it will be better to provide a script in bin directory, similar to what e.g. ts_MTAirCompressor has (or ts_cRIOpy). scripts are then defined in pyproject.toml. Consider calling the script m1m3_lut_update (remove .py from script name).